### PR TITLE
Add: additional entry with summary of video-info brand

### PR DESCRIPTION
**Pull request by TrueBrain**

### CODE REVIEW

File: analysis/__main__.py [hunk 1]
```diff
@@ -106,6 +107,19 @@ def summarize_setting(summary, version, seconds, path, data):
             data = data.split(",")[0].split("(")[0].strip()
             data = data.replace("@TM@", "(TM)").replace("@R@", "(R)").replace("@C@", "(C)")
 
+            if "nvidia" in data.lower() or "geforce" in data.lower() or "quadro" in data.lower():
+                brand = "NVIDIA"
```
Comment (LordAro) [inline analysis/__main__.py:111]: Nvidia I believe is the correct capitalisation

File: analysis/__main__.py [hunk 2]
```diff
@@ -106,6 +107,19 @@ def summarize_setting(summary, version, seconds, path, data):
...
+            elif "intel" in data.lower():
+                brand = "Intel"
+            elif "amd " in data.lower() or "radeon" in data.lower():
+                brand = "AMD"
+            elif "apple" in data.lower():
+                brand = "Apple"
+            else:
+                brand = "Unknown"
```
Comment (LordAro) [inline analysis/__main__.py:119]: What happens when there's inevitably a large amount of cards that appear as unknown due to the summarising?

File: analysis/__main__.py (hunk 1 above)
Comment (TrueBrain) [reply analysis/__main__.py:111]: Do you have any source for that?

Both do all cards report NVIDIA, as is everything on https://www.nvidia.com/en-us/ . Can't find a single entry not in full caps.

File: analysis/__main__.py [hunk 3]
```diff
@@ -106,6 +107,19 @@ def summarize_setting(summary, version, seconds, path, data):
             data = data.split(",")[0].split("(")[0].strip()
             data = data.replace("@TM@", "(TM)").replace("@R@", "(R)").replace("@C@", "(C)")
 
+            if "nvidia" in data.lower() or "geforce" in data.lower() or "quadro" in data.lower():
+                brand = "NVIDIA"
+            elif "intel" in data.lower():
+                brand = "Intel"
```
Comment (LordAro) [inline analysis/__main__.py:113]: Mildly biased here, but maybe worth keeping distinction between intel onboard and intel discrete gpus?

File: analysis/__main__.py (hunk 2 above)
Comment (TrueBrain) [reply analysis/__main__.py:119]: huh? I don't understand what you mean here.

File: analysis/__main__.py (hunk 1 above)
Comment (LordAro) [reply analysis/__main__.py:111]: You are correct, I am wrong

File: analysis/__main__.py (hunk 3 above)
Comment (TrueBrain) [reply analysis/__main__.py:113]: If you want to know the details of the Intel entries, use the non-brand table :) I really don't want this to be maintenance heavy. This is simple ;)

File: analysis/__main__.py (hunk 2 above)
Comment (LordAro) [reply analysis/__main__.py:119]: Arbitrarily grouping everything that doesn't fit the above conditonals into "unknown" doesn't feel like a good solution

Comment (TrueBrain) [reply analysis/__main__.py:119]: And as always, such comments are not helping me at all. Why doesn't it feel good? What do you need for it to do feel good? Do you have suggestions? Work with me here :D

**Comment by LordAro** [comment]
Oh wait, have I misunderstood this? It's an additional table, rather than instead of? If so, carry on :)

**Pull request merged**